src/controllers/attachments/index.ts

```diff
@@ -1,32 +1,120 @@
-import { Controller, Get, Route } from 'tsoa'
+import { UploadedFile, Request, Controller, Get, Route, Path, Post, Header, Response } from 'tsoa'
 import { Logger } from 'pino'
+import express from 'express'
 
 import { logger } from '../../lib/logger'
-import Database from '../../lib/db'
+import Database, { Models, Query } from '../../lib/db'
 import type { Attachments } from '../../models'
+import { BadReqeust, NotFound } from '../../lib/error-handler'
```

Comment:
`BadReqeust` typo

Reply:
ha... I'm embarrassed. To be fair when I type I do trust my muscle memory. Sometimes I like how I replicate typo across without being aware.